cedar-policy-cli/sample-data/tiny_sandboxes/json_policy/entity.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
Is it possible to reuse existing JSON files?

Reply:
It would, but all the `tiny_sandbox` examples are self contained, so I opted not to.